src/components/atoms/Avatar/Avatar.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const AvatarBase = styled.div`
+  width: ${(props) => props.size}rem;
+  height: ${(props) => props.size}rem;
+`;
+
+const AvatarImg = styled.img`
+  width: 100%;
+  height: 100%;
+
+  ${(props) =>
+    props.shape === 'circle' && {
+      borderRadius: '50%',
+    }}
+
+  ${(props) =>
+    props.shape === 'rounded' && {
+      borderRadius: '1.8rem',
+    }}
+`;
```

Comment:
I recommend extracting all the CSS styling to another file i.e. `Avatar.styled.js`, there export the styles and, in this file use them.

Reply:
If we extract the style from this file we can have a pattern of where styles are written and where logic business is written.